capi/geos_c.h.in

```diff
@@ -3635,6 +3658,69 @@ extern GEOSGeometry GEOS_DLL *GEOSOffsetCurve(const GEOSGeometry* g,
 
 ///@}
 
+
+/* ====================================================================== */
+/** @name Coverages
+* Functions to work with coverages represented by lists of polygons
+* that exactly share edge geometry.
+*/
+///@{
+
+/**
+* Analyze a coverage (represented as a collection of polygonal geometry
+* with exactly matching edge geometry) to find places where the
+* assumption of exactly matching edges is not met.
+*
+* \param input The polygonal coverage to access,
+*        stored in a geometry collection.
+* \return A value of 1 for a valid coverage, 0 for invalid and -1 for
+*         a processing error.
```

Comment:
Will this return -1 if any member of the collection is something other than a polygon / multipolygon?

Reply:
Yes